packages/main/src/plugin/experimental-feature-feedback-handler.ts

```
@@ -147,4 +155,73 @@ export class ExperimentalFeatureFeedbackHandler {
+      if (!featureGitHubLink || configuration.disabled) return;
+
+      // Compare timestamp of each experimental feature
+      const date = new Date();
+      if (configuration.remindAt && configuration.remindAt > date.getTime()) return;
+      const featureName = formatName(key);
+      const footerMarkdownDescription: string = `:button[fa-thumbs-up]{command=openExternal args='["${featureGitHubLink}"]'} :button[fa-thumbs-down]{command=openExternal args='["${featureGitHubLink}"]'}`;
+      const options: RemindOption[] = ['Remind me tomorrow', 'Remind me in 2 days', `Don't show again`];
+
+      this.messageBox
+        .showMessageBox({
+          title: `Share Your Feedback`,
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)